lib/player.js

```diff
@@ -3895,6 +3895,7 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
       // Try using the uri extension.
       const extension = shaka.media.ManifestParser.getExtension(uri);
       mimeType = {
+        'sbv': 'text/x-subviewer',
```

Comment:
It looks like the extension is `sub`.

Reply:
This format is used mainly on YouTube and as far as I know it is a subset. What is intended to implement is the support for this, in the description there is a link and an example of this. The extension we must use is sbv.